src/components/common/CategoryTabBar/index.tsx

```diff
@@ -39,12 +40,13 @@ const CategoryTabBar = ({ tabClick, tabState, tabList }: CategoryTabBarProps) =>
 export default CategoryTabBar;
 
 const Wrapper = styled.div`
+  z-index: ${Z_INDEX.TabBar};
```

Comment:
```suggestion
  z-index: ${Z_INDEX.Header};
```
TabBar는 화면 하단에 그 탭바라서 값은 같지만 키를 바꿀게요